### DISCUSSION TITLE
Possible errors in GitHub action help documentation

### DISCUSSION THREAD

**Discussion by FarEastSea** [community/community#62188] [Actions]
### Select Topic Area

Question

### Feature Area

Issues

### Body

在GitHub action 文档中说明了可以在if语境下省略${{}}表达式语法，使用表达式语法会让表达式被认定为字符串。但是在接下来的众多命令示例中，多次出现if: ${{ }}格式，这让我感觉前后矛盾，让我很迷茫。

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @FarEastSea , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one. 

For the future, you are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects.
